zerver/lib/message.py

```diff
@@ -1190,9 +1194,13 @@ def get_huddle_users(recipient_id: int) -> str:
         # TODO: Add support for alert words here as well.
         is_mentioned = (row["flags"] & UserMessage.flags.mentioned) != 0
         is_wildcard_mentioned = (row["flags"] & UserMessage.flags.wildcard_mentioned) != 0
+        is_direct_mentioned = (row["flags"] & UserMessage.flags.mentioned_me_directly) != 0
         if is_mentioned:
             mentions.add(message_id)
+        if is_direct_mentioned:
+            mentions_me_directly.add(message_id)
```

Comment:
you never initialized these (`mentions_me_directly` and `wildcard_mentions`) sets. go to line 1108 and add them under the `mentions: Set[int] = set()`

its too far up so i can't add the suggestion, but it should just look like:
```python
mentions: Set[int] = set()
mentions_me_directly: Set[int] = set()
wildcard_mentions: Set[int] = set()
```

Reply:
@yadhiramorales  @smallChenn